### ISSUE TITLE
Background-color not reset in continuous SPAN tags

### ISSUE THREAD

**Issue by seanliang** [daohoangson/flutter_widget_from_html#155]
```
import 'package:flutter/material.dart';
import 'package:flutter_widget_from_html/flutter_widget_from_html.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) => MaterialApp(
        title: 'Test',
        home: Column(
          crossAxisAlignment: CrossAxisAlignment.start,
          children: <Widget>[
            HtmlWidget(
              '<span style="color:#ff0;background-color:#00f;">==Span1==</span><br><span style="color:#f00;">**Span2**</span>',
            ),
            HtmlWidget(
              '<span style="color:#ff0;background-color:#00f;">==Span1==</span><span style="color:#f00;">**Span2**</span>',
            ),
          ],
        ),
      );
}
```
If the span tags are continuous, the background-color will be rendered on each span.

**Comment by daohoangson** [comment]
Thank you for the bug report.

**Comment by daohoangson** [comment]
I have just released version `0.3.3` with the fix for this. Can you try upgrading?

**Comment by seanliang** [comment]
Confirmed, thanks!
